src/consensus/network_event.rs

```diff
@@ -79,9 +79,21 @@ pub struct OnlinePayload {
 #[allow(clippy::large_enum_variant)]
 #[derive(Serialize, Deserialize, PartialEq, Eq, PartialOrd, Ord, Hash, Clone)]
 pub enum AccumulatingEvent {
-    /// Vote to start a DKG instance
+    /// Genesis event. This is output-only event.
+    Genesis {
```

Comment:
Agree almost wholly, but this thing is tricky. This is not really an event, neither is it a command, but at the same time it is kind of both. We can try thinking about how to express this better, but as we are trying to move routing away from parsec anyway, so at some point this whole thing will go away. So not sure it's worth it? 

Or maybe we can just do something simple for now: rename the event-like variants to past-tense verbs and the command-like variants to imperative-tense verbs?

Reply:
Does that then lean towards 2 enums? one for events and one for commands?

Reply:
Yeah, if you allow me some time I'm going to look at the names and the actual contexts of these and try come up with a good enough solution for now.

From what I have seen before, I think there is a possible cmd/event separation even here, if only some things are straightened out (we'll see if that holds true, and if there's not too much straightening to do).

So, basically, I think this could be the case:
> Does that then lean towards 2 enums? one for events and one for commands?

But will say more when I have fully seen what is meant by this:
> This is not really an event, neither is it a command, but at the same time it is kind of both.

Reply:
OK. As I said, I'm sure it's worth spending time on this as it'll all go away eventually. On the other hand, in the post-parsec world some of these will become normal messages and so we would still need to name them. So maybe worth to spend a little bit of time on this, just not too much?